### ISSUE TITLE
Add a script to find cases have been dispatched to BVA that shouldn't have been certified

### ISSUE THREAD

**Issue by alex** [department-of-veterans-affairs/caseflow#13]
After speaking with Jed, @plusjeff and I realized that we may need more than just the two scripts that Alex created. Here is what we are trying to get:

1. A list of all cases that have been dispatched to BVA that should not have been certified
2. A list of all cases that are awaiting certification but are missing files 
3. A list of all cases that are awaiting certification that have all their files and are awaiting a form 8

I believe that 2 and 3 are already solved, but 1 will require the use of the Mail Action field in VACOLS.

**Comment by alex** [comment]
Two reports we need:

- Cases in the seam
  - Certified to the board
  - Not activated
  - marked as anything besides VBMS (TODO: hybrid)
  - Have an efolder
  - Report fields
    - AoJ
    - Type (original, post-remand, etc.)
    - BFKEY
    - Mismatched documents
    - file_type (`tivbms`)

- Cases that should be returned to AoJ
  - Certified to the board
  - Not activated
  - VBMS case
  - Mismatched document dates

  - Report field
    - AoJ
    - Type
    - BFKEY
    - Mismatched documents

**Comment by alex** [comment]
Code I need to write:

- reporting harness
  - Execute VACOLS query
  - In parallel over each result: run VBMS check
  - write out to file
    - extract cells per row for report

**Comment by KHarshawat** [comment]
It would be useful to know and comment on the age of cases that were trapped in the seam. This could be calculated as time since the Notice of Disagreement was filed. This is a lower priority than the other parts of the script but if it is relatively easy to implement then we should add it.

**Comment by KHarshawat** [comment]
On second thought, a more useful date may actually be the number of days an appeal was trapped in the seam. This could be calculated by the date since the date was initially certified (Cert Date).

**Comment by paultag** [comment]
I'd also be interested in a net total of days in the seam
